core/chains/evm/txmgr/evm_tx_store_test.go

```diff
@@ -816,6 +816,12 @@ func TestORM_FindTransactionsConfirmedInBlockRange(t *testing.T) {
 		assert.Equal(t, etxes[0].Sequence, etx_8.Sequence)
 		assert.Equal(t, etxes[1].Sequence, etx_9.Sequence)
 	})
+
+	t.Run("return no error when no transactions in range found", func(t *testing.T) {
```

Comment:
Out of curiosity was this error before if we didn't find a transaction in the range?

Reply:
We don't. Bad wording, should just say returned empty txs